models/lineItem.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Created by patrickhalton on 8/19/15.
+ */
+
+var db          = require('../config/db'),
+    mongoose    = require('mongoose');
+
+var lineItemSchema = mongoose.Schema({
+
+    title : String,
+    content : String,
+    section_id : {type : Number, default : 0},
+    created_at : {type : Date, default: Date.now},
+    updated_at : {type : Date, default: Date.now}
+
+});
+
+var lineItemModel = mongoose.model('lineItems', lineItemSchema);
+
+
+exports.lineItemSection = function(liName,liContent){
+
```

Comment:
This is a Line Item File, you do not need to restate it is a line item in the function name. This will end up looking like the following in use.
```
var lineItem = require('./models/lineItem.js'); 
lineItem.lineItemSection("Branding", "Some info about Branding"); 
```

First off, I as a co-dev of yours have no idea what .linItemSection() should do. Does that Add a section to a line item? It should be the line item that is added to the section. Choose more descriptive names, that are more concise.

I'll refactor one of these files, please apply the changes to the other schema related files.